### ISSUE TITLE
NTR:Frontal polymicrogyria

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#4561]
label:Frontal polymicrogyria (synonym: ) ID NA parent Polymicrogyria def Polymicrogyria with anterior > posterior gradient extending from frontal poles posteriorly to precentral gyrus and inferiorly to frontal operculum [20301504] COST:neuromig,

**Comment by pnrobinson** [comment]
From the definition you provide, I think this is a synonym of 
https://hpo.jax.org/app/browse/term/HP:0006821 (Polymicrogyria, anterior to posterior gradient).

Would it be correct to relabel this term and add the above definition?

**Comment by Renske-Oegema** [comment]
Do you mean adding frontal pmg with Polymicrogyria, anterior to posterior gradiënt as a synonym?

**Comment by pnrobinson** [comment]
I think that you proposed we delete the term "Polymicrogyria, anterior to posterior gradient" and then create a term called "Frontal polymicrogyria". However, these appear to be synonyms to me and so it is easiest just to revise the term "Polymicrogyria, anterior to posterior gradient" by changing its label and adding the new definition?

**Comment by Renske-Oegema** [comment]
And add Frontal pmg as a synonym?

**Comment by pnrobinson** [comment]
@Renske-Oegema yes, both labels would be listed as synonyms. Is this correct? That is
Frontal polymicrogyria is a synonym of Polymicrogyria, anterior to posterior gradient

**Comment by Renske-Oegema** [comment]
OK
